### PR TITLE
[TACHYON-820] Implement RawTableMaster

### PR DESCRIPTION
**Pull request by uronce-cc**
https://tachyon.atlassian.net/browse/TACHYON-820

### CODE REVIEW

File: servers/src/main/java/tachyon/master/next/filesystem/FileSystemMaster.java [hunk 1]
```diff
@@ -344,12 +364,27 @@ public FileBlockInfo getFileBlockInfo(long fileId, int fileBlockIndex)
...
     // TODO: metrics
     synchronized (mInodeTree) {
-      mInodeTree.createPath(path, 0, recursive, true);
-      return true;
+      try {
+        mInodeTree.createPath(path, 0, recursive, true);
+      } catch (BlockInfoException bie) {
+        // Since we are creating a directory, the block size is ignored, no such exception should
+        // happen.
+        Throwables.propagate(bie);
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/next/filesystem/FileSystemMaster.java]: What does this line do, and why is it required to do it this way?

**Comment by gpang** [comment]
Thanks @uronce-cc ! I just had a minor question about Throwables.

File: servers/src/main/java/tachyon/master/next/filesystem/FileSystemMaster.java (hunk 1 above)
Comment (uronce-cc) [inline servers/src/main/java/tachyon/master/next/filesystem/FileSystemMaster.java]: It's equal to `new RuntimeException(bie)`, the codebase seems to prefer `Throwables.propagate(bie)`.

**Comment by gpang** [comment]
LGTM

**Pull request merged**